src/layouts/SiteLaunchPad/components/SiteLaunchCheckList.tsx

```diff
@@ -0,0 +1,369 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import {
+  Box,
+  Center,
+  Icon,
+  Table,
+  TableContainer,
+  Tbody,
+  Td,
+  Thead,
+  Tr,
+  Text,
+  TextProps,
+  Skeleton,
+} from "@chakra-ui/react"
+import { Button, Checkbox, Link } from "@opengovsg/design-system-react"
+import { useState } from "react"
+
+import { useSiteLaunchContext } from "contexts/SiteLaunchContext"
+
+import { BxCopy, BxLifeBuoy } from "assets"
+import {
+  DNSRecord,
+  SITE_LAUNCH_TASKS,
+  SITE_LAUNCH_TASKS_LENGTH,
+} from "types/siteLaunch"
+
+import { SiteLaunchPadBody } from "./SiteLaunchPadBody"
+import { SiteLaunchPadTitle } from "./SiteLaunchPadTitle"
+
+export const SiteLaunchChecklistTitle = (): JSX.Element => {
+  const title = "Complete these tasks to launch"
+  const subTitle =
+    "The following list has been generated based on information provided on your domain"
+  return <SiteLaunchPadTitle title={title} subTitle={subTitle} />
+}
+
+const getTextProps = (index: number, tasksDone: number): TextProps => {
+  const disabled = tasksDone >= index + 1 || tasksDone < index - 1
+  if (disabled) {
+    return {
+      color: "interaction.support.disabled",
+    }
+  }
+  return { as: "b" }
+}
```

Comment:
Just checking, is the previous element also supposed to be in bold text?

Reply:
nope, this is an intentional design choice to unbold if it is not impt (ie user is not at that current step at that point of time)

Reply:
eg. 
![Uploading Screenshot 2023-06-14 at 6.09.02 PM.png…]()

Reply:
eg 
<img width="603" alt="Screenshot 2023-06-14 at 6 09 02 PM" src="https://github.com/isomerpages/isomercms-frontend/assets/42832651/7f6aec41-b404-4e4c-9f3a-8cfbe55aa638">